cmd/mr_test.go

```diff
@@ -5,25 +5,64 @@ import (
 	"io/ioutil"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 	"strings"
 	"testing"
 
 	"github.com/acarl005/stripansi"
 	"github.com/stretchr/testify/require"
 )
 
+func cleanupMR(t *testing.T, targetRepo string, cmdDir string, MRtitle string) {
+	openMRcmd := exec.Command("sh", "-c", fmt.Sprintf("%s mr list %s | grep -m1 \"%s\" | cut -c2- | awk '{print $1}'", labBinaryPath, targetRepo, MRtitle))
+	openMRcmd.Dir = cmdDir
+	b, err := openMRcmd.CombinedOutput()
+	if err != nil {
+		t.Log(string(b))
+		t.Fatal(err)
+	}
+
+	openMR, err := strconv.Atoi(strings.TrimSpace(string(b)))
+	if err != nil {
+		t.Log(string(b))
+		return
+	}
+
+	if openMR <= 0 {
+		// no open MRs
+		return
+	}
+
+	// close the existing MR
+	closeMRcmd := exec.Command("sh", "-c", fmt.Sprintf("%s mr close %d\n", labBinaryPath, openMR))
```

Comment:
@prarit I think I found the issue: you're not passing the `targetRepo` for this command, so the default one is being used, which isn't the same where the MR is.
If you check the test results the cleanup code tries to delete the MR from projectID `4181224` (zaquestion/test), but the correct should be `5694926` (lab-testing/test)

Reply:
Oh geez. That's braindead of me.

Reply:
Hahaha.
But why not just use "closeMR" it does exactly the same as this command, only additionally checks for the "close" string, which should cause no harm?

Reply:
Ok, I'm going to manually delete the MR 4702 and re-run the test.